initial_height: 0.6
limited_height: 2.0

# for uav_response
torelance_z: 0.03

# for simple_explore
pose_z_diff: 0.1
tvoc_weight: 0.2

# for grid explore
grid_size: 0.1